src/layouts/ReviewRequest/Dashboard.tsx

```diff
@@ -0,0 +1,167 @@
+                  variant="clear"
+                  aria-label="link to pull request"
+                  onClick={onCopy}
+                />
+              </PopoverTrigger>
+              <PopoverContent
+                bg="background.action.alt"
+                _focus={{
+                  boxShadow: "none",
+                }}
+                w="fit-content"
+              >
+                <PopoverArrow bg="background.action.alt" />
+                <PopoverBody>
+                  <Text textStyle="body-2" color="text.inverse">
+                    Link copied!
+                  </Text>
+                </PopoverBody>
+              </PopoverContent>
+            </Popover>
+          </HStack>
+          <Spacer />
+          <ApprovalButton />
+        </Flex>
+        <SecondaryDetails
+          requestor={requestor}
+          reviewers={reviewers}
+          reviewRequestedTime={new Date()}
+        />
+      </VStack>
+      <Box pl="9.25rem" pr="2rem">
+        <RequestOverview items={changedItems} allowEditing />
+      </Box>
+    </Box>
+  )
+}
+
+// NOTE: Utility component exists to soothe over state management
+const ApprovalButton = (): JSX.Element => {
+  const [isApproved, setIsApproved] = useState(false)
+
+  return (
+    <MenuDropdownButton
+      colorScheme={isApproved ? "success" : "primary"}
+      mainButtonText={isApproved ? "Approved" : "Pending Review"}
+      variant="solid"
+    >
+      <MenuDropdownItem onClick={() => setIsApproved(false)}>
+        <Text textStyle="body-1" textColor="text.body" w="100%">
+          Pending review
+        </Text>
+      </MenuDropdownItem>
+      <MenuDropdownItem onClick={() => setIsApproved(true)}>
+        <Text textStyle="body-1" textColor="text.success" w="100%">
+          Approved
+        </Text>
+      </MenuDropdownItem>
+    </MenuDropdownButton>
+  )
+}
+
+interface SecondaryDetailsProps {
+  requestor: string
+  reviewers: string[]
+  reviewRequestedTime: Date
+}
+const SecondaryDetails = ({
+  requestor,
+  reviewers,
+  reviewRequestedTime,
+}: SecondaryDetailsProps) => {
+  const { date, time } = getDateTimeFromUnixTime(
+    reviewRequestedTime.getTime() / 1000
+  )
+  return (
+    <VStack spacing="0.5rem" align="flex-start">
+      <Text textStyle="caption-2" textColor="text.helper">
+        {`Review requested by ${requestor} on ${date} ${time}`}
+      </Text>
+      <HStack spacing="0.75rem">
+        <Text textStyle="caption-2" textColor="text.helper">
+          Reviewers
+        </Text>
+        <Box>
+          {reviewers.map((name) => {
+            const initials = extractInitials(name)
+            return (
+              <Avatar
+                border="1px solid white"
+                mr="-0.25rem"
+                bg="primary.100"
+                name={initials}
+                textStyle="caption-1"
+                textColor="primary.400"
+                size="sm"
+              />
+            )
+          })}
```

Comment:
Very minor nit, depends on what design says: The overlapping of avatars is different from the Figma design.
* Figma: Left avatar overlaps over the right avatar
<img width="248" alt="Screenshot 2022-09-30 at 11 05 05" src="https://user-images.githubusercontent.com/27919917/193181412-8f7088c9-8bc4-4be4-a184-cbbcfd75dfd3.png">

* Here: Right avatar overlaps over the left avatar
<img width="130" alt="Screenshot 2022-09-30 at 11 05 32" src="https://user-images.githubusercontent.com/27919917/193181455-50c9660e-1433-42ab-b9fb-2bbd7e8b0299.png">

Reply:
ooh oops, think i saw wrongly HAHAHAH